python_ta/util/monad.py

```diff
@@ -29,14 +29,17 @@ def __str__(self):
         
     def bind(self, fn):
         return fn(self.value)
+
         
-        
-def failable_map(fn : Callable[[TypeVar('T')], Failable], lst : List[Failable]): 
-    # TODO: allow arbitrary containers like tuples
+def failable_map(fn : Callable[[TypeVar('T')], Failable], itr: Iterable[Failable]):
+    if isinstance(itr, list):
```

Comment:
Don't worry about making this check, just call `list(itr)` regardless of what its type is.